README.md

```diff
@@ -1,135 +1,221 @@
+
+:warning: Note that each request you send to Metrilo is limited to **5MB** in size.
+
+:warning: Most of the time the data you send will override any existing data. For example, if you make a call about product *A* with categories *C1* and *C2* and then you make another call only with category *C3*, then the first two categories will be removed from the product. However, sometimes the data will be merged (e.g. adding a product's options). If the data is merged, it will be explicitly mentioned in the documentation.
+
+### Tracking library installation
+
+Insert the following script tag in the page you want to track (we recommend inserting it in the `<head>` tag):
+```
+<script type="text/javascript" src="https://trk.mtrl.me/tracking.js?token=<YOUR_PROJECT_TOKEN>"></script>
+```
+
+Replace `<YOUR_PROJECT_TOKEN>` with the API token from within the `Settings -> Installation` page in your Metrilo project.
+
+:warning: Make sure to remove the `<` and `>` symbols. So, for instance, if your token was `1234asdf`, you would need to put in this snippet:
+```
+<script type="text/javascript" src="https://trk.mtrl.me/tracking.js?token=1234asdf"></script>
+```
+
+When you have successfully loaded the tracking library, you will have access to a `window.metrilo` object on your page.
+
+:warning: **You should add the script tag on every page that you want to track!**
+
+### Sending tracking events
+
+A tracking event is an action that a customer of yours does on your website - such as viewing a page, adding an item to their cart, ordering a product, etc. These actions are saved in the customer's current session and are the base of which Metrilo does its magic.
+
+These are the functions you need to call on the `window.metrilo` object based on what a customer does on your website.
+
+| Whenever a customer                 | you need to call                  |
+| :---------------------------------- | :-------------------------------- |
+| enters their email on your website  | [identify](#identify)             |
+| visits a page                       | [viewPage](#viewPage)             |
+| visits an article page              | [viewArticle](#viewArticle)       |
+| visits a category page              | [viewCategory](#viewCategory)     |
+| visits a product page               | [viewProduct](#viewProduct)       |
+| types something in the search bar   | [search](#search)                 |
+| adds a product to their cart        | [addToCart](#addToCart)           |
+| removes a product from their cart   | [removeFromCart](#removeFromCart) |
+| visits the checkout page            | [checkout](#checkout)             |
+
+We also provide two more functions that you can use at any point you like - [applyTags](#apply-tags) and [customEvent](#custom-event).
```

Comment:
these two events are not obvious. If I check what are the lib events, I will skim through the table and not look bellow it. Maybe, include them in the table, too

Reply:
👍